device-huawei-c8813q-configs/var/lib/environment/compositor/droid-hal-device.conf

```diff
@@ -0,0 +1,5 @@
+# Config for huawei/c8813q
+HYBRIS_EGLPLATFORM=fbdev
```

Comment:
please test if fingerterm works. if not, most likely this flag is the reason, and it anyhow needs to be removed (will be fixed in next HADK pdf)